modules/tensor_mechanics/test/tests/finite_strain_elastic/tests

```diff
@@ -3,53 +3,80 @@
   [../]
   [./fake_plastic]
     type = 'Exodiff'
     input = 'finite_strain_fake_plastic.i'
     exodiff = 'finite_strain_fake_plastic_out.e'
+    issues = '#9212 #7555'
+    design = 'source/Materials/ComputeMultiPlasticityStress tensor_mechanics/VolumetricLocking.md'
+    requirement = 'The ComputeMultiPlasticityStress class shall, when supplied with no plastic models, reduce to and produce the solely elastic stress solution for a finite strain fomulation, using the TensorMechanicsMaster action.'
   [../]
   [./fake_plastic_Bbar]
     type = 'Exodiff'
     input = 'finite_strain_fake_plastic.i'
     exodiff = 'finite_strain_fake_plastic_out.e'
     cli_args = 'GlobalParams/volumetric_locking_correction=true'
     prereq = 'fake_plastic'
+    issues = '#9212 #8235'
+    design = 'source/Materials/ComputeMultiPlasticityStress tensor_mechanics/VolumetricLocking.md'
+    requirement = 'The ComputeMultiPlasticityStress class shall, when supplied with no plastic models, reduce to and produce the solely elastic stress solution for a finite strain fomulation, using the volumetric locking correction b-bar formulation.'
   [../]
   [./rotation_new]
     type = 'Exodiff'
     input = 'elastic_rotation_test.i'
     exodiff = 'elastic_rotation_test_out.e'
+    issues = '#4716 #7555'
+    design = 'source/Materials/ComputeFiniteStrainElasticStress.md syntax/Modules/TensorMechanics/Master/index.md'
+    requirement = 'The ComputeFiniteStrainElasticStress class shall compute the elastic stress based on a finite strain fomulation and then follow the stress as the mesh is rotated 90 degrees in accordance with Kamojjala et al. Engineering with Computers (31) 2015 using the TensorMechanicsMaster action.'
   [../]
   [./rotation_new_Bbar]
     type = 'Exodiff'
     input = 'elastic_rotation_test.i'
     exodiff = 'elastic_rotation_test_out.e'
     cli_args = 'GlobalParams/volumetric_locking_correction=true'
     prereq = 'rotation_new'
+    issues = '#4716 #8235'
+    design = 'source/Materials/ComputeFiniteStrainElasticStress.md tensor_mechanics/VolumetricLocking.md'
+    requirement = 'The ComputeFiniteStrainElasticStress class shall compute the elastic stress based on a finite strain fomulation and then follow the stress as the mesh is rotated 90 degrees in accordance with Kamojjala et al. Engineering with Computers (31) 2015 using the volumetric locking correction b-bar formulation.'
   [../]
   [./eigen_sol]
     type = 'Exodiff'
     input = 'finite_strain_elastic_eigen_sol.i'
     exodiff = 'finite_strain_elastic_eigen_sol_out.e'
+    issues = '#4716 #7555'
+    design = 'source/Materials/ComputeFiniteStrainElasticStress.md source/Materials/ComputeFiniteStrain.md syntax/Modules/TensorMechanics/Master/index.md'
+    reuirement = 'The ComputeFiniteStrainElasticStress class shall compute the elastic stress for a finite strain formulation found with the eigen-strain solution on a unit 3D cube in a Cartesian system using the TensorMechanicsMaster action.'
```

Comment:
'found with the eigen-strain solution' -> 'using a direct eigensolution to perform the polar decomposition of stretch and rotation'